crates/uv/tests/tool_list.rs

```diff
@@ -111,7 +111,7 @@ fn tool_list_bad_environment() {
         .success();
 
     // Remove the python interpreter for black
-    fs::remove_dir_all(tool_dir.path().join("black/bin").as_os_str()).unwrap();
+    fs::remove_dir_all(tool_dir.path().join("black/bin").as_os_str())?;
```

Comment:
Minor note that there's no real preference between the two of these in our test suite. `unwrap()` is nice because you get a line number on failures. `?` is nice because it's succinct.